caffe2/CMakeLists.txt

```diff
@@ -91,6 +91,9 @@ if(NOT BUILD_ATEN_ONLY)
   if (BUILD_CAFFE2_OPS)
     add_subdirectory(operators)
     add_subdirectory(operators/rnn)
+    if (USE_FBGEMM)
+      add_subdirectory(operators/quantized/server)
```

Comment:
non-default operator implementations are conventionally stored in `caffe2/contrib`, e.g. `caffe2/contrib/fbgemm`

Reply:
@Maratyszcza @harouwu I thought we agreed to put these in quantized/server dir. Anyway this would be a big change considering the deadline but we can discuss and do it later